### ISSUE TITLE
is this still open source? 

### ISSUE THREAD

**Issue by ChuckBartowski91** [rignaneseleo/SlimSocial-for-Facebook#238]
Hello, why the github code is outdated from 3 months? Is because is developed on flutter? Where the code can be looked at? Tapping on codemagic icon on the main page returns not found.

But more important, is this still safe to use?

**Comment by rignaneseleo** [comment]
Yes. The app is open source and in active development again. The shipped code is the Flutter app in `SlimSocial_for_Facebook/`, and a Kotlin rewrite is in progress in PR #298.

Please use version **26.09.03+128** or later. Open a new issue if the problem comes back.
